### ISSUE TITLE
XRF Updates

### ISSUE THREAD

**Issue by coolacid** [g4klx/ircDDBGateway#26]
Not sure pistar.uk is the correct place to get updates for the XRF reflectors - Lastest update removed XRF021 from the list, which is still listed as active at http://xrefl.net/ (And the dashboard is still active) 

I'd suggest using ftp://dschost2.w6kd.com/DExtra_Hosts.txt as the base for updates.

**Comment by K2DLS** [comment]
I don't think that the W6KD list is a good choice.  It has not been updated in a year!  So it looks like that one is not being actively maintained.
```
#>>Downloaded from W6KD host file server
#>>Last updated 27 APR 2018 by W6KD
```
Another choice could be the K7VE list:

http://ar-dns.net/dextra
